new evaluations.html

```diff
@@ -62,6 +68,16 @@
        
 
     <button class="button button1">Submit</button>
+<<<<<<< Updated upstream
```

Comment:
this is a git conflict that has not been managed and it will break your new evaluations.html visuals

Reply:
nu am dat merge,may be it is the problem:)

Reply:
most likely, but those problems have to be taken care of :)

Reply:
ok,sry